### PR TITLE
Hyperparameter opt lab: Fix learning rate range

### PR DESCRIPTION
**Pull request by vvolhejn**
The range was such that to<from, also the comment didn't match the range. I took a wild guess as to which one is the intended one.

### CODE REVIEW

**Comment by foxik** [comment]
The wild guess is the correct one, thanks!
